includes/api/podcast.php

```diff
@@ -244,4 +269,15 @@ private function getCategoryName($category_key)
             }
         }
     }
+
+    private function getLanguageName($language_key) 
+    {
+        $language = \Podlove\Locale\locales();
+        foreach($language as $key => $val) {
+            if ($key == $language_key) {
+                return $val;
+            }
+        }
+    }
```

Comment:
Hier fänd ich einen Fallback gut, falls aus irgend einem Grund ein unbekannter Sprachcode gespeichert ist. Sonst wird glaube ich `NULL` zurückgegeben. Könnte Probleme machen, wenn der Aufrufer von einem String ausgeht.

```suggestion
        }
        
        return "";
    }
```

Reply:
Habe ich so geändert, das Verhalten ändert sich dadurch aber etwas. Es wird jetzt immer ein String zurückgegeben auch wenn der Wert nie gesetzt wurde.